erpnext/patches/v14_0/dynamic_fiscal_years_for_irs_1099.py

```diff
@@ -0,0 +1,270 @@
+import frappe
+
+
+def execute():
+	# pull from_year and to_year dynamically in irs_1099 print format
+	irs_1099_print_format = frappe.get_doc("Print Format", "IRS 1099 Form")
+	irs_1099_print_format.html = """
```

Comment:
this won't update in new installations, right?

Is there no way to just change .json file and migrate it (like doctypes/report etc)

Reply:
ignore this

Reply:
`frappe.reload_doc("regional", "print_format", "irs_1099_form", force=True)`

works with custom formats too.